lib/msal-node/apiReview/msal-node.api.md

```diff
@@ -449,6 +449,7 @@ export type ManagedIdentityIdParams = {
 //
 // @public
 export type ManagedIdentityRequestParams = {
+    claims?: string;
```

Comment:
My understanding is that the claims also needs to be passed to the managed identity endpoint. @gladjohn can you confirm that?

Reply:
oops. Yea, I'll have to add them to the endpoint.

Reply:
done

Reply:
MI v1 endpoint do not take any claims, the ask is to simply bypass cache with claims is passed to MSAL.